packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt

```diff
@@ -33,60 +37,90 @@ import kotlinx.coroutines.withTimeout
  */
 internal class DataGripMongoDbDriver(
     private val project: Project,
-    private val dataSource: LocalDataSource
+    private val dataSource: LocalDataSource,
 ) : MongoDbDriver {
     private val gson = Gson()
+    private val jsonWriterSettings = JsonWriterSettings.builder()
+.outputMode(JsonMode.EXTENDED)
+.indent(false)
+.build()
+
+    private fun Bson.toJson(): String = this.toBsonDocument().toJson(
+            jsonWriterSettings,
+        )
+
+    override suspend fun serverUri(): URI = URI.create(dataSource.url!!)
```

Comment:
This `URI` won't have a `host` property if it doesn't parse as a valid URI (e.g. multiple hosts). Should we really be using `URI` here?

Reply:
That's a good point. Maybe we can use the ConnectionString class from the Java driver.